pkg/storage/tracing/tracing.go

```diff
@@ -74,8 +74,8 @@ func (t *tracingStorage) CleanupStorage(ctx context.Context, spec models.InputSo
 	defer func() {
 		dur := stopwatch()
 		log.Ctx(ctx).Debug().
-			Dur("duration", dur).
-			Object("spec", &spec).
+			Dur("Duration", dur).
+			Str("Alias", spec.Alias).
```

Comment:
Why is this being changed?

Reply:
I checkedout code from this PR before it was merged https://github.com/bacalhau-project/bacalhau/pull/3667